docker/Dockerfile.server

```
@@ -2,9 +2,24 @@
 
 FROM mcr.microsoft.com/windows/server:ltsc2022
 
+# OCI labels for metadata
+LABEL org.opencontainers.image.title="Windows Server LTSC 2022 Image pre-installed with uv and VC Redist"
+LABEL org.opencontainers.image.description="A Windows Server image with the uv python package and project manager from astral  installed, Python version 3.9-3.13 pre-installed, and including the Visual C++ Redistributable."
+LABEL org.opencontainers.image.version="0.0.2"
```

The version should be the same as the `uv` (and image tag) version
```suggestion
LABEL org.opencontainers.image.version="${UV_VERSION}"
```

---

Reviewer suggestion adopted and incorporated in commit 087e34076735b42bedc516abebf11506294b2b42.